doc/content/integrations/storage/configuration.md

```diff
@@ -25,6 +25,16 @@ Initialize the configured database with:
 $ ttn-lw-stack storage-db init
 ```
 
+{{< note >}}
+If you are using TimescaleDB, the following configuration options are also available:
+
+```
+--timescaledb.chunk-time-interval
+--timescaledb.enable-retention-policy
```

Comment:
Options are `--timescaledb.enable-retention-policy` and `--timescaledb.enable-retention-policy=false` (to disable).

The retention period is configured with the `--timescaledb.retention-days` flag (see below).

Reply:
Note that enabling this flag will have an immediate effect and start deleting upstream messages that are older than the specified number of days.